api/src/iot/tests/test_import_utils.py

```diff
@@ -327,7 +327,7 @@ def test_import_sensor_other_type(self, sensor_data):
     def test_import_sensor_regions(self, sensor_data):
         # check that we can import as "mobile" sensor
         owner = import_utils.import_person(sensor_data.owner)
-        sensor_data.location = import_utils.Regions("Centrum,Oost")
+        sensor_data.location = import_utils.Regions("Centrum;Oost")
```

Comment:
Hier moet je de setting gebruiken, of in ieder geval expliciet overriden (django.test.override_settings):

```
@override_settings(IPROX_SEPARATOR=";")
```

Reply:
Goede

- [ ] Gebruik settings scheidingsteken